CHANGELOG

```diff
@@ -1,3 +1,7 @@
+1.2.0 (2020-04-24)
+-----------------------------------
```

Comment:
Missing a few `----`

Reply:
thanks, fixed now

Reply:
I don't think we were missing. Usually we aligh dashes with release name, like:

```
1.2.0 (2020-04-24)
------------------
```

But also we usually call it `Version 1.2.0`, which got lost after extraction, hence they're shorter: https://github.com/snowplow-incubator/iglu-server/blob/master/CHANGELOG#L1-L4

Reply:
Oops indeed I got confused by the wrong alignment, my mistake !